invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -21,6 +21,23 @@
     }
   }
 
+  /*Field Label*/
+  .ui.form .field .form-group label{
```

Comment:
Also here, I think is better to apply a class to the fields you want to use that css rule! Also, now that the rdm theme is merged maybe you can put these rules in the `form.overrides` file instead of here! That way, we can start changing the `.ui.xxx` rules...